shipit.production.yml

```diff
@@ -5,7 +5,7 @@ dependencies:
   override:
     - curl -fsSL https://get.pnpm.io/install.sh | SHELL=`which bash` bash -
     - bash -i -c "pnpm install"
-
+    - echo 'registry "https://registry.npmjs.org/"' | tee .npmrc
```

Comment:
I'm very confused. So we are using pnpm to call changesets, then they are running npm internally, and finally the yarn registry is used? 😄 

According to [this comment](https://github.com/yarnpkg/yarn/issues/2935#issuecomment-355292633), we should update the registry in .yarnrc instead.

Reply:
The registry configuration is independent from the package manager you use and it can be configured globally or locally. And then there are package managers like Yarn that decide to come up with their own conventions, like having their own `.yarnrc` configuration file 🤦🏼. I'll update that file too just in case.